src/wallet/wallet.cpp

```diff
@@ -5150,13 +5150,13 @@ CAmount CWallet::GetLegacyBalance(const isminefilter& filter, int minDepth) cons
     return balance;
 }
 
-void CWallet::AvailableCoins(vector<COutput>& vCoins,
+std::vector<COutput> CWallet::AvailableCoins(
                              const std::optional<int>& asOfHeight,
                              bool fOnlyConfirmed,
                              const CCoinControl *coinControl,
                              bool fIncludeZeroValue,
                              bool fIncludeCoinBase,
-                             bool fOnlySpendable,
+                             isminefilter filter,
```

Comment:
To implement my suggestion:
```suggestion
                             isminefilter spendabilityFilter,
                             bool fOnlySpendable,
```